onchain/rollups/test/foundry/util/LibBytes.sol

```diff
@@ -11,6 +11,21 @@ library LibBytes {
         return address(bytes20(keccak256(data)));
     }
 
+    /// @notice Generate an address array from a byte array.
+    /// @param data The byte array
+    /// @param length The expected address array length
+    /// @return The address array
+    function hashToAddressArray(
+        bytes memory data,
+        uint256 length
+    ) internal pure returns (address[] memory) {
+        address[] memory array = new address[](length);
+        for (uint256 i; i < length; ++i) {
+            array[i] = address(bytes20(keccak256(abi.encodePacked(data, i))));
```

Comment:
Since returning `IInputRelay[]` is not useful for general purpose, I remove this function from the library. Instead, it's directly implemented [in the test contract](https://github.com/cartesi/rollups-contracts/blob/877b6cc15a90b7885c69ab1ead36af4596afd0f6/onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol#L110).